airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -218,6 +218,17 @@ def name(self) -> str:
 
     def list(self, fields) -> Iterable:
         yield from self.read(partial(self._api.get, url=self.url))
+    
+    def _filter_dynamic_fields(self, records: Iterable) -> Iterable:
+        """Skip certain fields because they are too dynamic and change every call (timers, etc),
+        see https://github.com/airbytehq/airbyte/issues/2397
+        """
+        for record in records:
+            if isinstance(record, Mapping) and "properties" in record:
+                for key in list(record["properties"].keys()):
+                    if key.startswith("hs_time_in"):
+                        record["properties"].pop(key)
+            yield record
```

Comment:
As you remove these `hs_time_in` fields from ignored fields in `acceptance-test-config`?

Reply:
We move it back and forth
As for me we don't need to remove data `hs_*` fields from response
Some customers complained about this

https://github.com/airbytehq/airbyte/issues/8055

Reply:
doc string of reverted back `_filter_dynamic_fields` function points on issue #2397
which "says" that we don't need to filter-out `hs_time_*` fields now
we DO filter-out this fileds on `acceptance-test` level

Reply:
I've removed this method again in 42b1f42 and can confirm it unit tests and acceptance tests still pass.